src/scrapers/pelitorrentScraper.ts

```diff
@@ -0,0 +1,151 @@
+import { load } from 'cheerio';
+import { slugify } from '../lib/strings';
+import Scraper from './scraper';
+import { Magnet, ScraperRequest } from '../interfaces';
+
+const SOURCE = 'PeliTorrent';
+const DEFAULT_URL = 'https://torrenflix.com';
```

Comment:
As the site has been moved to `https://torrenflix.com`, should we rename the scraper to "TorrentFlix"?